src/main.c

```diff
@@ -26,6 +26,7 @@ int main(int argc, char **argv) {
   start_listener(st);
 
   destroy_state(st);
+  noraw();
```

Comment:
I think this is irrelevant. `raw()` only overwrites ctrl-c, ctrl-z, etc. Even if it somehow fixes the issue, when the editor crashes because of a runtime error, it won't be executed anyway.